app/assets/stylesheets/layout.scss

```diff
@@ -396,6 +399,7 @@ The copy box can't be invisible/hidden or copy fails but opacity works
   height: 20px;
 }
 #_hiddenCopyText_ { height: 0px; width: 0px; opacity: 0; padding: 0px; margin: 0px; }
+
```

Comment:
The two styles above this new line break are the ones that the comment is referring to about:
```
Styles for the user output page to handle copy/pasting text
The copy box can't be invisible/hidden or copy fails but opacity works
```
Since the subsequent styles have nothing to do with the user output page I wanted them separated by a break